### PR TITLE
Main

### LINKED ISSUE #4: Make account app complete
**Describe the solution you'd like**
Make three different types of user Farmer, Adviser and Researcher using userprofile model which inherited form user model

### PR DESCRIPTION
**Pull request by Mayankraj23**
### Description
Created the models for main app.

Fixes #4

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by maintainers)

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have commented my code or provided relevant documentation
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/main/models.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from src.oauth.models import Farmer
```
Comment (Mayankraj23) [inline src/main/models.py]: by making it just oauth.models, django is showing 'unresolved reference'

File: src/main/models.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    primary_crop = (
+        (1, 'Rice'),
+        (2, 'Wheat'),
+        (3, 'Pulses'),
+    )
+
+    """
+    Various Model Fields
+    """
+    farmer = models.ForeignKey(Farmer, on_delete= models.CASCADE)
```
Comment (Mayankraj23) [inline src/main/models.py]: I want to use all the fields of the Farmer model, so why make it One to One

File: src/main/models.py (hunk 1 above)
Comment (hirenchalodiya1) [reply src/main/models.py]: its error of pycharm not of django

File: src/main/models.py (hunk 2 above)
Comment (hirenchalodiya1) [reply src/main/models.py]: foreinkey means one farmer_detail model have many farmer

File: src/main/models.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mayankraj23) [reply src/main/models.py]: then how to resolve that error, coz if its not resolved, my server won't run

[the remark this answers is not in the document]
Comment (hirenchalodiya1) [reply src/main/models.py]: have you applied migrations??

**Pull request merged**